hedera-node/hedera-app/src/main/java/com/hedera/node/app/info/DiskStartupNetworks.java

```diff
@@ -335,6 +337,10 @@ private static void archiveIfPresent(@NonNull final Configuration config, @NonNu
         }
     }
 
+    private static void archiveIfPresent(@NonNull final Configuration config, @NonNull final String... segments) {
```

Comment:
After this I think we should also fix the error you got in `V0540RosterSchema.restart` if the `config.txt` is not present to not fail

Reply:
ah got it, will do 👍

Reply:
Added